src/ajax.js

```diff
@@ -71,6 +71,9 @@ export function ajaxBuilder(timeout = 3000) {
             }
           }
         };
+        x.ontimeout = function () {
+          utils.logError('  xhr timeout after ', x.timeout, 'ms');
```

Comment:
any reason for the spaces?

Reply:
since there's also an `xhr error` console message just before this I figured the indentation makes it a little clearer that they are related